xfce/exo/Portfile

```diff
@@ -4,11 +4,11 @@ PortSystem      1.0
 PortGroup       perl5 1.0
 
 name            exo
-version         4.16.4
+version         4.18.0
```

Comment:
Same, please update to the latest version.

Reply:
As far as I understand, 4.19.x versions are development versions for what will ultimately become the 4.20.x series.